platformcomponents/desktop/attachment-file-content-grid.json

```diff
@@ -0,0 +1,12 @@
+{
+  "attachment-file-content-grid": {
```

Comment:
I cannot find attachment-file-content-grid in the component library

Reply:
Just pushed commit with links for Content tab components.
`Added Figma links for Content file uploads`
Please check again.
@nataliadelmar

Reply:
I see... we have agreed not to add any token that is not official in the component library. Otherwise, it becomes a mess of links. I have left a comment in that Figma to ask Cass what should we do about this one (I tagged you on it :) ).

Reply:
This component, named at the time of the comment `attachment-file-content-grid` has been renamed to `content-file-card-item` and has been added to the main Figma Components Catalog.
Changing this conversation to Resolved.